backend-beeyond/src/main/java/at/htl/beeyond/dto/UserListDto.kt

```diff
@@ -0,0 +1,17 @@
+package at.htl.beeyond.dto
+
+import at.htl.beeyond.entity.User
+import at.htl.beeyond.validation.Checks
+import at.htl.beeyond.validation.Exists
+import javax.validation.GroupSequence
+
+@GroupSequence(value = [Checks.UserListExists::class, UserListDto::class])
+data class UserListDto(
+    @field:Exists(
+        entity = User::class,
+        fieldName = "name",
+        groups = [Checks.UserListExists::class]
+    ) var users: List<String>?
```

Comment:
Also is this list really optional?. This List should never be empty thus you need `@NotNull` and `@Size(min = 1)`